_overviews/scala3-migration/tooling-tour.md

```diff
@@ -107,7 +107,7 @@ So far the relevant rules are:
 - [Procedure Syntax](https://scalacenter.github.io/scalafix/docs/rules/ProcedureSyntax.html)
 - [Explicit Result Types](https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes.html)
 - Value Eta-Expansion: `fix.scala213.ExplicitNullaryEtaExpansion` in [scala/scala-rewrites](https://github.com/scala/scala-rewrites/blob/main/rewrites/src/main/scala/fix/scala213/ExplicitNullaryEtaExpansion.scala)
-- Parentheses Around Lambda Parameter: `fix.scala213.ParensAroundLambda` in [ohze/scala-rewrites](https://github.com/ohze/scala-rewrites/blob/dotty/rewrites/src/main/scala/fix/scala213/ParensAroundLambda.scala)
+- Parentheses Around Lambda Parameter: `migrate.ParensAroundParam` in [scalacenter/scala3-migrate plugin](https://github.com/scalacenter/scala3-migrate/blame/ebb4a4087ed11899b9010f4c75eb365532694c0a/scalafix/rules/src/main/scala/migrate/ParensAroundParam.scala#L9)
```

Comment:
although this is an improvement in that it's at least pointing to the right place, this is an internal rule by the looks rather than a published one - it may be that it's not usable as-is.

Reply:
Is it used via the migration plugin? Then this advertises the feature for the plugin.

Reply:
Having gone back and read this document again, I think this line just needs removing.  As it's totally irrelevant to anyone migrating - as the transformation is covered by the -rewrite scalac flag mentioned above, and users won't care about rules they can't use directly.  I will push a change to this file.